src/main/java/com/simibubi/create/content/contraptions/Contraption.java

```diff
@@ -1167,6 +1169,14 @@ public void addBlocksToWorld(Level world, StructureTransform transform) {
 					.isEmpty())) {
 					if (targetPos.getY() == world.getMinBuildHeight())
 						targetPos = targetPos.above();
+					if(state.hasBlockEntity()){
+						CompoundTag tag = block.nbt();
+						if(tag != null){
+							BlockEntity blockEntity = BlockEntity.loadStatic(targetPos, state, tag);
+							if (blockEntity instanceof Container container)
```

Comment:
`Container` is probably (definitely, the Placard for example already doesn't implement it) too limited.

Reply:
The issue i have found is that most methods access the block entity using the Position and Level directly, so they don't work in this case where the block entity is not actually anywhere. I was thinking to maybe use Item Requirements from the schematicanon implementation, it seems to complement this approach (would not work with placards until https://github.com/Creators-of-Create/Create/pull/8550). To me seems like a weird way to patch it and i'm not sure is desirable in the codebase, but gonna give it a try if noone stops me.

Reply:
I was adding exceptions for create containers, but maybe SmartBlockEntity.destroy() is enough for the majority of cases